deluge/ui/gtkui/preferences.py

```diff
@@ -978,6 +980,21 @@ def _on_combo_proxy_type_changed(self, widget):
             else:
                 self.builder.get_object(entry).hide()
 
+    def _on_entry_proxy_host_paste_clipboard(self, widget):
+        text = ''.join(gtk.clipboard_get().wait_for_text().split())
```

Comment:
We are primarily concerned with trailing newlines causing issues so start and end is fine.